xsl/webhelp-common.xsl

```diff
@@ -639,9 +654,27 @@ border: none; background: none; font-weight: none; color: none; }
     <xsl:template name="webhelpheader.logo">
 	<a href="index.html">
 	<img style='margin-right: 2px; height: 59px; padding-right: 25px; padding-top: 8px' align="right"
-	    src='{$webhelp.common.dir}images/logo.png' alt="{$brandname} Documentation"/>
+	    src='{$webhelp.common.dir}images/logo_Rudder.png' alt="{$brandname} Documentation"/>
 	</a>
     </xsl:template>
+    
+    
+    <xsl:template name="otherlinks">
+    <div id="otherlinks">
+      <span>Version: <strong>2.11 (ESR)</strong> | <a href="http://www.rudder-project.org/doc-3.0/">3.0</a> | 
+                                                   <a href="http://www.rudder-project.org/doc-3.1/">3.1</a> |
+                                                   <a href="http://www.rudder-project.org/doc-3.2/">3.2</a></span>
+      <span>Download as: <a href="/rudder-doc-2.11/rudder-doc.pdf">epub</a> | <a href="/rudder-doc-2.11/rudder-doc.pdf">pdf</a></span>
```

Comment:
We have to be careful for these links because they won't work on the embedded version in 3.2.
We could:
* put pdf and epub in the webhelp folder for Rudder >= 3.2 (as webhelp becomes the default) and redirect /rudder-doc-3.2/rudder-doc.pdf to /doc-3.2/rudder-doc.pdf in public docs, to keep the same layout everywhere
* Have a special case to use different links for embedded docs